tests/api.c

```diff
@@ -47209,6 +47209,201 @@ static int test_wolfSSL_BIO_tls(void)
     return EXPECT_RESULT();
 }
 
+
+static int test_wolfSSL_BIO_datagram(void)
+{
+    EXPECT_DECLS;
+#if !defined(NO_BIO) && defined(WOLFSSL_DTLS) && defined(WOLFSSL_HAVE_BIO_ADDR) && defined(OPENSSL_EXTRA)
+    int ret;
+    SOCKET_T fd1 = 0, fd2 = 0; /* SOCKET_T is unsigned on Windows */
```

Comment:
Use `SOCKET_INVALID` for invalid socket values.

Reply:
fixed